### PR TITLE
Updates .po files

### LINKED ISSUE #158: Rebuild translation files
We need to update translation files with makemessages, then we'll be able to work on translations with rosetta. 

https://docs.djangoproject.com/en/4.0/ref/django-admin/#makemessages

### PR DESCRIPTION
**Pull request by victorphoenix3**
Fixes #158 
Signed-off-by: Jayati Shrivastava <gaurijove@gmail.com>

### CODE REVIEW

**Review by 2ynn — APPROVED** [review]
Thank you!

File: saskatoon/sitebase/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -17,244 +17,769 @@ msgstr ""
...
+msgstr ""
 
-#~ msgid "Welcome"
-#~ msgstr "Bienvenu(e)"
+#: sitebase/templates/registration/login.html:104
+msgid "Login"
+msgstr ""
 
-#~ msgid "Change password"
+#~ msgid "hehe"
```
Comment (2ynn) [inline saskatoon/sitebase/locale/fr/LC_MESSAGES/django.po:774]: I'm confused, where is the "hehe" coming from?

Comment (victorphoenix3) [reply saskatoon/sitebase/locale/fr/LC_MESSAGES/django.po]: @2ynn , I accidentally pushed the test patch, I have updated the PR now. Please review.

**Comment by tristanlatr** [comment]
Thanks a lot @victorphoenix3!

**Pull request merged**